OpenRA.Game/Server/Server.cs

```diff
@@ -1163,6 +1163,12 @@ public Session.Client GetClient(Connection conn)
 			return LobbyInfo.ClientWithIndex(conn.PlayerIndex);
 		}
 
+		/// <Remarks> Does not check if client is admin </Remarks>
+		public bool CanKickClient(Session.Client client)
```

Comment:
This appears to be only used once, and not even in this class?

Reply:
Yes, we need access to worldPlayers variable which is supposed to be private on the server

Reply:
I see.